### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by rchikatw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,13 +30,21 @@
...
 ## Steps to migrate provider
 - Clone the github repository
-- Run script using ./migrate.sh -d [env for consumer addon [-dev]/[-qe]], When you have dev addon installed ex: 
+- Run script using ./migrate.sh -provider <oldClusterID> <newClusterID> -d [env for consumer addon [-dev]/[-qe]], When you have dev addon installed ex: 
 ```
- ./migrate.sh -d -dev
+ ./migrate.sh -provider <oldClusterID> <newClusterID> -d -dev
+```
+- If you dont pass env for addon the script will consider as prod addon.
+- MTSRE will use ocm-backplane and hence they don't need to pass -d option. Like
```
Comment (rewantsoni) [inline README.md]:
```suggestion
- If we have access to ocm-backplane, we do not require the -d option, we can run the script as: 
```

File: README.md [hunk 2]
```diff
@@ -31,13 +30,21 @@
...
 ```
- ./migrate.sh -d -dev
+ ./migrate.sh -provider <oldClusterID> <newClusterID> -d -dev
+```
+- If you dont pass env for addon the script will consider as prod addon.
+- MTSRE will use ocm-backplane and hence they don't need to pass -d option. Like 
+```
+    ./migrate.sh -provider <oldClusterID> <newClusterID>
+```
+- After the provider migration completed, The script will print the commands to run for each consumer, Run them one by one or in separate terminals to speed up the migration. Like
```
Comment (rewantsoni) [inline README.md]:
```suggestion
- After the provider migration completed, The script will print the commands to run for each consumer.
- Run them one after the other or in separate terminals to speed up the migration. The template for consumer migration command would be:
```

**Comment by rewantsoni** [comment]
@rchikatw I think we can also update the usage section of each script in this PR.

File: backupResources.sh [hunk 3]
```diff
@@ -7,14 +7,11 @@ usage() {
...
+    1. kubectl: ${link[kubectl]}
+    2. OCM: ${link[ocm]}
     2. clusterID for the cluster
 
-  USAGE: "./backupResources.sh"
-
-  To install kubectl & ocm refer:
-  1. kubectl: ${link[kubectl]}
-  2. ocm: ${link[ocm]}
+  USAGE: "./backupResources.sh <oldClusterID> -d"
```
Comment (rewantsoni) [inline backupResources.sh]:
```suggestion
  USAGE: "./backupResources.sh <oldClusterID> [-d]"
```
As -d is optional we should add it like this

File: backupResources.sh [hunk 4]
```diff
@@ -7,14 +7,11 @@ usage() {
   Takes the backup of required resources for restoring/migrating provider cluster.
 
   Requirements:
-    1. kubectl, ocm installed.
+    1. kubectl: ${link[kubectl]}
+    2. OCM: ${link[ocm]}
     2. clusterID for the cluster
```
Comment (rewantsoni) [inline backupResources.sh]:
```suggestion
    2. clusterID for the Old cluster
```

File: migrate.sh [hunk 5]
```diff
@@ -7,10 +7,15 @@ usage() {
...
   Requirements:
-    1. kubectl, ocm installed.
-    2. clusterID for the clusters
+    1. kubectl: ${link[kubectl]}
+    2. yq: ${link[yq]}
+    3. jq: ${link[jq]}
+    4. clusterID for the clusters
+    5. ROSA: ${link[rosa]}
+    6. OCM: ${link[ocm]}
+    7. aws: ${link[aws]}
```
Comment (rewantsoni) [inline migrate.sh]:
```suggestion
    1. kubectl: ${link[kubectl]}
    2. yq: ${link[yq]}
    3. jq: ${link[jq]}
    4. ROSA: ${link[rosa]}
    5. OCM: ${link[ocm]}
    6. aws: ${link[aws]}
    7. clusterID for the clusters
```

File: migrate.sh [hunk 6]
```diff
@@ -7,10 +7,15 @@ usage() {
...
 
-  USAGE: "./migrate.sh [-d] [env for consumer addon [-dev]/[-qe]]"
+  USAGE: "./migrate.sh -provider <oldClusterID> <newClusterID> -d [env for consumer addon [-dev]/[-qe]]"
```
Comment (rewantsoni) [inline migrate.sh]:
```suggestion
  USAGE: 
  provider: "./migrate.sh -provider <oldClusterID> <newClusterID> [-d] [env for consumer addon [-dev]/[-qe]]"
  consumer: "./migrate.sh -consumer <consumerClusterID> <storageProviderEndpoint> <storageConsumerUID from old provider> [-d] [env for consumer addon [-dev]/[-qe]]"
```

File: README.md (hunk 1 above)
Comment (rchikatw) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (rchikatw) [reply README.md]: Done

File: backupResources.sh (hunk 3 above)
Comment (rchikatw) [reply backupResources.sh]: Done

File: backupResources.sh (hunk 4 above)
Comment (rchikatw) [reply backupResources.sh]: Done

File: migrate.sh (hunk 5 above)
Comment (rchikatw) [reply migrate.sh]: Done

File: migrate.sh (hunk 6 above)
Comment (rchikatw) [reply migrate.sh]: Done

**Pull request merged**